src/App.js

```diff
@@ -15,11 +15,16 @@ import Links from './components/links';
 import Cluster from './components/cluster';
 import FAQ from './components/faq';
 import Banner from './components/banner';
+import Popup from './components/popup';
 /* import PatientDB from './components/patientdb';*/
 
+import {useTranslation} from 'react-i18next';
+
 const history = require('history').createBrowserHistory;
 
 function App() {
+  const {t} = useTranslation();
```

Comment:
const { translate } = useTransaltion();

`translate` would be more meaningful than just `t`

Reply:
@puru0019

Hook is from react-i18next and we can't change it directly https://react.i18next.com/latest/usetranslation-hook

Reply:
if `translate` is a must this might help
```js
const { t: translate } = useTransaltion();
```

Reply:
IMHO, @KrishnaPravin, I know aliasing is possible but doing it everywhere, or wrapping it in another hook isn't a clean approach.

they are naming function as 't' because it's less distractive provides better DX, ctrl clicking on t goes to useTranslate that anyway says that this is a translate function plus most react translation libraries do it same way.

https://www.npmjs.com/package/react-translate
https://react.i18next.com/latest/usetranslation-hook

Reply:
understood👍